introduction.md

```diff
@@ -31,6 +31,18 @@ and environments ([Java][java-tls], [PHP Curl][php-curl-tls],
 You can inspect [Swedbank Pay's TLS and cipher suite][ssllabs] support at
 SSL Labs. Support for HTTP/2 in our APIs is being investigated.
 
+{% include alert.html type="informative" icon="info" body= "We encourage you to
```

Comment:
I don't think this needs to be an alert, since everything said in the `introduction` is as important as this. Give it its own title, such as `Postel's Robustness Principle`. That's also going to fix the unrendered link (Markdown doesn't work in the `alert`'s `body`).

Reply:
Please remove the `include alert.html`, make this a proper section and give it a header as suggested. :)

Reply:
Huh? I pulled it to VS code and did the changes there. I thought I commited the changes and pushed them. I'll do it again 😄